src/main/kotlin/no/nav/syfo/syfosmvarsel/nysykmelding/NySykmeldingService.kt

```diff
@@ -0,0 +1,66 @@
+package no.nav.syfo.syfosmvarsel.nysykmelding
+
+import no.nav.syfo.model.ReceivedSykmelding
+import no.nav.syfo.syfosmvarsel.LoggingMeta
+import no.nav.syfo.syfosmvarsel.domain.OppgaveVarsel
+import no.nav.syfo.syfosmvarsel.metrics.NY_SM_MOTTATT
+import no.nav.syfo.syfosmvarsel.metrics.NY_SM_VARSEL_OPPRETTET
+import no.nav.syfo.syfosmvarsel.util.innenforArbeidstidEllerPaafolgendeDag
+import no.nav.syfo.syfosmvarsel.varselutsending.VarselProducer
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.time.LocalDateTime
+import java.util.*
+
+private val log: Logger = LoggerFactory.getLogger("no.nav.syfo.syfosmvarsel")
+
+// Henger sammen med tekster i mininnboks: http://stash.devillo.no/projects/FA/repos/mininnboks-tekster/browse/src/main/tekster/mininnboks/oppgavetekster
+const val OPPGAVETYPE = "0005"
+
+fun opprettVarselForNySykmelding(
+    receivedSykmelding: ReceivedSykmelding,
+    varselProducer: VarselProducer,
+    tjenesterUrl: String,
+    loggingMeta: LoggingMeta
+) {
+    try {
+        log.info("Mottatt ny sykmelding med id {}, $loggingMeta", receivedSykmelding.sykmelding.id, *loggingMeta.logValues)
+        NY_SM_MOTTATT.inc()
```

Comment:
Her vil vi øke telleren selv om kallet feiler. Da teller vi f.eks retries flere ganger. 

Kan evt. flytte den til etter vi har sendt varselet. Men da er det jo ikke vits med to tellere...

Reply:
Godt poeng! 😅 Er det egentlig noen vits med to tellere..? Jeg lagde det sånn opprinnelig slik at vi kunne fange opp hvis det stoppet opp slik at vi oppdager hvis sykmelding inn != varsel som opprettes, men jeg vet ikke hvor sannsynlig det er at skjer uten at vi merker det..?

Reply:
Nei jeg vet ikke hvor realistisk det er at vi fanger opp en liten diff - og at varlser opprettet da blir den mest aktuelle metrikken.